purchase_request_to_rfq/models/purchase_request.py

```diff
@@ -18,7 +18,7 @@ class PurchaseRequestLine(models.Model):
     @api.depends('purchase_lines')
     def _compute_is_editable(self):
         super(PurchaseRequestLine, self)._compute_is_editable()
-        for rec in self.filtered(lambda p: p in self and p.purchase_lines):
+        for rec in self.mapped('purchase_lines'):
```

Comment:
`self.filtered(lambda p: p in self and p.purchase_lines)` returns all `purchase.request.line` that are in `self` and have at least one linked `purchase.order.line`.
`self.mapped('purchase_lines')` returns all `purchase.order.line` linked to every `purchase.request.line` in `self`.

The right modification is : `self.filtered(lambda p: p.purchase_lines)`